source/js/components/pulse-profile/pulse-profile.jsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { getText } from "../petition/locales";
+
+const PulseProfile = ({ profile }) => {
+  const profileUrl = "https://www.mozillapulse.org/profile/";
+  return (
+    <div className="tw-grid tw-grid-cols-4 tw-gap-x-3 tw-gap-y-2 tw-border-t tw-border-black">
+      {/* Image */}
+      <a
+        href={`https://www.mozillapulse.org/profile/${profile.profile_id}`}
+        className="tw-block tw-col-span-1 tw-relative tw-min-h-[160px] tw-h-[100%] tw-w-full tw-overflow-hidden"
+      >
+        <img
+          src={
+            profile.thumbnail
+              ? profile.thumbnail
+              : "/static/_images/fellowships/headshot/placeholder.jpg"
+          }
+          className="tw-w-auto tw-h-full tw-absolute tw-left-50 tw-top-50 tw-min-w-full tw-object-cover"
+          alt="Headshot"
+        />
+      </a>
+
+      {/* Right card  */}
+      <div className="tw-col-start-2 tw-col-span-3 tw-flex tw-flex-col">
+        {/* Card top */}
+        <div className="tw-flex tw-flex-row tw-justify-between tw-items-start tw-mt-2">
+          <a
+            className="tw-text-lg tw-mb-1 tw-font-sans tw-font-normal tw-text-black"
+            href={`${profileUrl}${profile.profile_id}`}
+          >
+            {profile.name}
+          </a>
+
+          {/* Social Icons */}
+          <div className="tw-flex tw-flex-row tw-space-x-2 tw-mt-[7px]">
+            {profile.twitter && (
+              <a
```

Comment:
I believe this should get rid of the errors from the node tests:
`  38:15  error  Anchors must have content and the content must be accessible by a screen reader  jsx-a11y/anchor-has-content`

```suggestion
              <a href={profile.twitter}>
                  <i className="twitter twitter-glyph small"/>
              </a>
```

However, we may need some CSS to remove the text-decoration:underline on hover, as when I hovered over a twitter glyph on my 16 in Macbook, there seemed to be a very small underline to the right of the icon

Reply:
Awesome thanks for looking into this, I have made the change now :)